### PR TITLE
BigAnimal: updates to PGD CLI commands

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

https://enterprisedb.atlassian.net/browse/UPM-23155

spec: https://enterprisedb.atlassian.net/wiki/spaces/PD/pages/3452207113/pgd

### CODE REVIEW

**Review by francoughlin — DISMISSED** [review]
I think it looks good!

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 1]
```diff
@@ -279,14 +410,27 @@ To restore a deleted cluster, use the `--from-deleted` flag in the command.
...
 
-Use the `pgd` commands to create, retrieve information on, and manage extreme-high-availability clusters. 
+## Managing extreme-high-availability clusters
+
+Use the BigAnimal `pgd` commands to create, retrieve information on, and manage extreme-high-availability clusters. 
 
+In addition to these BigAnimal `pgd` CLI commands, you can use [EDB Postgres Distributed CLI  commands](/pgd/latest/cli/) to manage your extreme-high-availability clusters. 
 
-### `pgd create` 
+!!!warning
```
Comment (francoughlin) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:420]: This entire procedure is appearing within the **Warning** box. Is that intentional?

**Review by francoughlin — APPROVED** [review]
I think this is fine for now, however I did have one question about the procedure appearing in the warning box.

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 1 above)
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:420]: nope, not intentional. thanks for checking that

**Review by francoughlin — DISMISSED** [review]
Looks good!

**Review by francoughlin — DISMISSED** [review]
Looks good, but I did have minor two questions, which can be answered at another time:
- Was the intent to change the name from extreme-high-availability to distributed high-availability everywhere, or just in the CLI section (for example, I see it in other places, such as in the Getting started section, "Creating an extreme-high-availability cluster").
- This is a big nit, but I had been looking at the marketing docs and they doo not hyphenate  high-availability. Probably leave that call to Betsy, if she has not already made it?

**Review by francoughlin — APPROVED** [review]
Yup, I think that looks fine

**Comment by drothery-edb** [comment]
Natalia approved on the ticket

**Pull request merged**